### PR TITLE
Fix EventLog remark

### LINKED ISSUE #18997: EventLog logging configuration clarification
In the section on windows event log, there is a sentence that reads:

> If EventLog log settings are specified, they default to LogLevel.Warning.

Should this be:
> If EventLog log settings are **not** specified, they default to LogLevel.Warning.

I am having an issue where the first statement is occurring.
I am specifying the LogLevel as Information, but it is still defaulting to Warning as per the documentation, which seems odd.

```
{
  "Logging": {
    "LogLevel": {
      "Default": "Debug"
    }
  },
  "EventLog": {
    "LogLevel": {
      "Default": "Information"
    }
  }
}
```

HostBuilder looks like this if it matters:
```
public static IHostBuilder CreateWebHostBuilder(string[] args) =>
            Host.CreateDefaultBuilder(args)
                .ConfigureWebHostDefaults(webBuilder =>
                {
                    webBuilder.UseStartup<Startup>()                       
                        .ConfigureLogging((hostingContext, logging) =>
                        {
                            logging.AddEventLog(options =>
                            {
                                options.SourceName = "ExampleApp";
                            });
                        })
                        .CaptureStartupErrors(true);
                });
```
Using core 3.1
Microsoft.Extensions.Logging.EventLog 3.1.5

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* Version Independent ID: 726e3bf1-f367-d733-8933-bccc04da0e16
* Content: [Logging in .NET Core and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1#bilp)
* Content Source: [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/logging/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by philip-reed**
Fixes #18997

Corrected typo regarding default log levels for EventLog.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=18998) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=18998) before we can accept your contribution.<br/><br/>:x: philip-reed [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=18998)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=18998) it.</sub>

**Comment by guardrex** [comment]
@philip-reed ... Sign the CLA if you agree with the terms. :point_up:

**Comment by guardrex** [comment]
Not sure tho if it should say "add the following" ... I'll make a small touch to that remark. Stand-by.

**Comment by guardrex** [comment]
ok .... I'm done **_mess'in around with ur PR_** now @philip-reed! :smile:

**Comment by philip-reed** [comment]
Ha, no worries.
Is there anything else I need to do?

**Comment by guardrex** [comment]
Nah ... looks good. I'll just wait for @serpent5 to look before merging it.

File: aspnetcore/fundamentals/logging/index.md [hunk 1]
```diff
@@ -561,14 +561,16 @@ To configure PerfView for collecting events logged by this provider, add the str
 
 ### Windows EventLog
 
-The `EventLog` provider sends log output to the Windows Event Log. Unlike the other providers, the `EventLog` provider does ***not*** inherit the default non-provider settings. If `EventLog` log settings are specified, they [default to LogLevel.Warning](https://github.com/dotnet/extensions/blob/release/3.1/src/Hosting/Hosting/src/Host.cs#L99-L103).
+The `EventLog` provider sends log output to the Windows Event Log. Unlike the other providers, the `EventLog` provider does ***not*** inherit the default non-provider settings. If `EventLog` log settings aren't specified, they [default to LogLevel.Warning](https://github.com/dotnet/extensions/blob/release/3.1/src/Hosting/Hosting/src/Host.cs#L99-L103).
 
-To log events lower than `Warning`, explicitly set the log level. For example, add the following to the *appsettings.json* file:
+To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. In the following example, the Event Log default log level is set to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index.md:566]:
```suggestion
To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. The following example sets the Event Log default log level to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```

File: aspnetcore/fundamentals/logging/index.md [hunk 2]
```diff
@@ -1459,12 +1461,14 @@ logging.AddEventLog();
 * `SourceName`: ".NET Runtime"
 * `MachineName`: The local machine name is used.
 
-Events are logged for [Warning level and higher](#log-level). To log events lower than `Warning`, explicitly set the log level. For example, add the following to the *appsettings.json* file:
+Events are logged for [Warning level and higher](#log-level). To log events lower than <xref:Microsoft.Extensions.Logging.LogLevel.Warning?displayProperty=nameWithType>, explicitly set the log level. In the following example, the Event Log default log level is set to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```
Comment (serpent5) [inline aspnetcore/fundamentals/logging/index.md:1464]:
```suggestion
Events are logged for [Warning level and higher](#log-level). The following example sets the Event Log default log level to <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>:
```

**Comment by guardrex** [comment]
Thanks @philip-reed :beers: and @serpent5 🐢.

**Pull request merged**